### ISSUE TITLE
Initiate the process analysis and incorporate it into the report

### ISSUE THREAD

**Issue by MadsHeilmann** [audio-engineer/aau-p2-article#25]
Begin the development of the process analysis such that the team can write it in parallel to the report and the code.

**Comment by audio-engineer** [comment]
@MadsHeilmann I will open a separate issue to improve the appendices as per https://latex-tutorial.com/latex-appendix/. The process analysis should be added there as well

**Comment by MadsHeilmann** [comment]
> @MadsHeilmann I will open a separate issue to improve the appendices as per https://latex-tutorial.com/latex-appendix/. The process analysis should be added there as well

Okay. I will close this one again then :)
